app/services/data_migrations/backfill_export_type.rb

```diff
@@ -4,28 +4,58 @@ class BackfillExportType
     MANUAL_RUN = false
 
     def change
-      data_exports = DataExport.all.where(export_type: nil)
+      ActiveRecord::Base.connection.execute('SET statement_timeout = 0')
+      data_exports = DataExport.where(export_type: nil)
```

Comment:
is it possible to not pull in the `data` column?
would this work?
```suggestion
      data_exports = DataExport.select(:id, :name, :export_type).where(export_type: nil)
```

Reply:
Yea this works